spec/features/see_reviews_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+feature "see reviews", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.